corehq/apps/reports/standard/deployments.py

```diff
@@ -80,60 +79,62 @@ def headers(self):
     def selected_app_id(self):
         return self.request_params.get(SelectApplicationFilter.slug, None)
 
+    @memoized
```

Comment:
10mins? Basically so that if you run the report a few times you can get the benefit of the cache. This is particularly important for large apps like ICDS which take a long time to load and 'parse'